polyfill/intersection-observer.js

```diff
@@ -714,8 +723,11 @@ IntersectionObserver.prototype._rootIsInDom = function() {
  * @private
  */
 IntersectionObserver.prototype._rootContainsTarget = function(target) {
-  return containsDeep(this.root || document, target) &&
-    (!this.root || this.root.ownerDocument == target.ownerDocument);
+  const rootDoc = isDoc(this.root) ? this.root : this.root.ownerDocument;
+  return (
+    containsDeep(this.root || document, target) &&
```

Comment:
Should this be `this.root || rootDoc`?

Reply:
is there a difference? if `this.root` is not set, then `document` and `rootDoc` should be the same thing right?

Reply:
updated

Reply:
Hmm. Yeah. You're right about this one. But I guess either way if you've already updated it.